script/travis-script

```diff
@@ -14,14 +14,26 @@ source "$HOME/.bashrc"
 
 mix compile
 
-# report test coverage to coveralls.io
+# report unit test coverage to coveralls.io
 # shellcheck disable=SC2037
-MIX_ENV=test mix coveralls.travis --exclude slow
+MIX_ENV=test mix coveralls.travis test/ts/unit --exclude slow
 
-# report test coverage to codecov
+# report unit test coverage to codecov
 # shellcheck disable=SC2037
 MIX_ENV=test mix do compile coveralls.json
 
+bash <(curl -s https://codecov.io/bash) -F unit
```

Comment:
clean up of coverage reports is required here; other wise, unit test coverage also gets merged with the integration test coverage.